etcdctl/READMEv3.md

```diff
@@ -281,8 +281,220 @@ foo
 bar
 ```
 
+### LEASE \<subcommand\>
+
+LEASE provides commands for key lease management.
+
+### LEASE GRANT \<ttl\>
+
+LEASE GRANT creates a fresh lease with a given time-to-live in seconds.
```

Comment:
with a server selected time-to-live in seconds that it is at least equal to the given one.